### ISSUE TITLE
Add alt text to call components that use MediaInput

### ISSUE THREAD

**Issue by spacedmonkey** [GoogleForCreators/web-stories-wp#5452]
## Feature Description

Found as part of https://github.com/google/web-stories-wp/issues/5249 . Accessibility tests on e2e tests failed because the image preview in the [MediaInput](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/components/form/media.js) component did not output an alt tag. 

<img width="1884" alt="Screenshot 2020-11-25 at 16 26 20" src="https://user-images.githubusercontent.com/237508/100255760-d8189e00-2f3b-11eb-9826-bd59d9d398b6.png">

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by miina** [comment]
Was fixed as part of the redesign.
